make/addons.mk

```diff
@@ -87,8 +87,7 @@ template-helm-repository: generate-mindthegap-repofile ## this is used by goreal
 
 .PHONY: list-images
 list-images:
```

Comment:
Why not leave the redirection? 

Users are already able to run the tool as is to get the output

Reply:
Using the make recipe builds and calls the tool with the correct flags which is just a bit simpler for the user.

I found this much more convenient to use when I was testing the changes I made and it feels more natural for the make target to list to stdout and allow the caller to redirect where they want. I think I saw @dkoshkin recommend the same thing on the original PR, which triggered me changing it to this - I prefer this approach as well.

Reply:
I made the original suggestion because it my mind it makes the make target more portable.
The caller can decide what to do with this output, write it out to file, sort it or filter it, etc.

Even when reviewing this PR, it made it slightly simpler calling the target and seeing the output in my terminal and not having to look up a file name.

Reply:
I think it’s a helpful thing when reviewing a pr like this but we need to add more steps to the goreleaser file. 

If you want to list the images you can create another target or run the tool directly.

Reply:
Why do you feel we need more steps in the goreleaser config? It seems to work fine in my tests as is

Reply:
I'm missing something - the change here is the only change from your pr in goreleaser config which calls the same target. No extra logic to your pr, just very minor tweaks in how make is called. Adding another target is unnecessary with how it works in this pr and doesn't require any extra logic in goreleaser config from what I can see.

Reply:
@faiq I understand your concern with more logic in the goreleaser file but it already needs to know the name of the file [here](https://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix/pull/900/files#diff-42e26dc67aed8aa3edb2472b4403288c1699fb6dc47419b9a475f0f224fe4689L34), shouldn't it also own writing to this file instead of having to keep track of it in 2 different files?

Reply:
I just don't think this target needs to print to stdout by default when there isn't a use case for this that will normally come up. I've simplified this in my PR as well 

https://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix/pull/822/commits/892c6e4bb57d2b11ef4da79fd7ca10b96afea28a

Reply:
Can we agree to disagree? A couple of us would like to go one way, one would like to go another. It feels like we're taking an unnecessary amount of time for a trivial change where there is no right or wrong answer, just a difference of opinion.

Reply:
ok ill die on another hill